gcloud/rest/auth/iam.py

```diff
@@ -0,0 +1,149 @@
+import json
+import threading
+from typing import Dict
+from typing import List
+from typing import Optional
+from typing import Union  # pylint: disable=unused-import
+
+import requests
+
+from .token import Token
+from .token import Type
+from .utils import encode
+
+
+API_ROOT_IAM = 'https://iam.googleapis.com/v1'
+API_ROOT_IAM_CREDENTIALS = 'https://iamcredentials.googleapis.com/v1'
+SCOPES = ['https://www.googleapis.com/auth/iam']
+
+
+class IamClient:
```

Comment:
Inherit from `object`?

Reply:
yup, a lint error caught this too. Thought I had fixed it, will be fixed in the next update to this PR.